eras/conway/impl/cardano-ledger-conway.cabal

```diff
@@ -69,7 +69,7 @@ library
         cardano-ledger-binary >=1.1,
         cardano-ledger-allegra >=1.1,
         cardano-ledger-alonzo ^>=1.4,
-        cardano-ledger-babbage >=1.1,
+        cardano-ledger-babbage >=1.4.1,
```

Comment:
I think we need to increase the patch of this then? In case it's not already increased after rebasing

Reply:
Looks like master has 1.6.1.1 right now, but depends on when this gets merged.